### ISSUE TITLE
`M-l` leads to crash

### ISSUE THREAD

**Issue by emacsomancer** [atlas-engineer/next#109]
I can use `C-l` fine to open a new url in the current tab. But `M-l` leads to Next completely freezing and needing to be xkill'ed (on my Arch machine) or crashing (on my GuixSD machine) with the following error displayed in terminal:

```
(next-gtk:6410): Gtk-CRITICAL **: 09:27:05.739: gtk_tree_path_to_string: assertion 'path != NULL' failed
Unhandled SB-INT:SIMPLE-PROGRAM-ERROR in thread #<SB-THREAD:THREAD "cl-cffi-gtk main thread" RUNNING
                                                   {1004655193}>:
  invalid number of arguments: 0

Backtrace for: #<SB-THREAD:THREAD "cl-cffi-gtk main thread" RUNNING {1004655193}>
0: (NEXT::SET-URL-NEW-BUFFER) [external]
1: (NEXT::CONSUME-KEY-SEQUENCE)
2: (GOBJECT::CALL-WITH-RESTARTS #<FUNCTION (LAMBDA (INTERFACE::WINDOW INTERFACE::EVENT) :IN INTERFACE:START) {22286A3B}> (#<GTK:GTK-WINDOW {10046B5673}> #S(GDK:GDK-EVENT-KEY :TYPE :KEY-PRESS :WINDOW #<GDK:GDK-WINDOW {1005713D03}> :SEND-EVENT NIL :TIME 36894816 :STATE (:MOD1-MASK) :KEYVAL 108 :LENGTH 1 :STRING "l" :HARDWARE-KEYCODE 46 :GROUP 0 :IS-MODIFIER 0)))
3: ((LAMBDA (GOBJECT::CLOSURE GOBJECT::RETURN-VALUE GOBJECT::COUNT-OF-ARGS GOBJECT::ARGS GOBJECT::INVOCATION-HINT GOBJECT::MARSHAL-DATA) :IN "/home/slade/common-lisp/quicklisp/dists/quicklisp/software/cl-cffi-gtk-20160208-git/gobject/gobject.signals.lisp") #.(SB-SYS:INT-SAP #X7FFFDC145AE0) #.(SB-SYS:INT-SAP #X7FFFE16A6100) 2 #.(SB-SYS:INT-SAP #X7FFFE16A61B0) #<unused argument> #<unused argument>)
4: ((LAMBDA (SB-ALIEN::ARGS-POINTER SB-ALIEN::RESULT-POINTER FUNCTION) :IN "/home/slade/common-lisp/quicklisp/dists/quicklisp/software/cl-cffi-gtk-20160208-git/gobject/gobject.signals.lisp") #<unavailable argument> #<unavailable argument> #<unavailable argument>)
5: ("foreign function: funcall_alien_callback")
6: ("foreign function: callback_wrapper_trampoline")
7: ("foreign function: #x201004DB")
8: ((LAMBDA NIL :IN GTK:ENSURE-GTK-MAIN))
9: ((LAMBDA NIL :IN BORDEAUX-THREADS::BINDING-DEFAULT-SPECIALS))
10: ((FLET SB-UNIX::BODY :IN SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE))
11: ((FLET "WITHOUT-INTERRUPTS-BODY-4" :IN SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE))
12: ((FLET SB-THREAD::WITH-MUTEX-THUNK :IN SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE))
13: ((FLET "WITHOUT-INTERRUPTS-BODY-1" :IN SB-THREAD::CALL-WITH-MUTEX))
14: (SB-THREAD::CALL-WITH-MUTEX #<CLOSURE (FLET SB-THREAD::WITH-MUTEX-THUNK :IN SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE) {7FFFE16A6D5B}> #<SB-THREAD:MUTEX "thread result lock" owner: #<SB-THREAD:THREAD "cl-cffi-gtk main thread" RUNNING {1004655193}>> NIL T NIL)
15: (SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE #<SB-THREAD:THREAD "cl-cffi-gtk main thread" RUNNING {1004655193}> NIL #<CLOSURE (LAMBDA NIL :IN BORDEAUX-THREADS::BINDING-DEFAULT-SPECIALS) {100465513B}> NIL)
16: ("foreign function: call_into_lisp")
17: ("foreign function: new_thread_trampoline")

unhandled condition in --disable-debugger mode, quitting
```

**Comment by jmercouris** [comment]
I'm sorry, but I can't provide support for GTK, Next is currently undergoing a major rewrite, and that includes the GTK version. I can't imagine why that is happening, but it is probably due to my poor understanding of GTK. I apologize for the problems!

**Comment by Ambrevar** [comment]
I'll look into it when I find time to work on Next. (Hopefully soon :p)

**Comment by watkinsr** [comment]
Same error for me as well

**Comment by Ambrevar** [comment]
This should be fixed with #137.  Feel free to re-open if the issue persists.
